routes/api.php

```diff
@@ -26,7 +26,7 @@
 
 // comment post request
 Route::post('comment', [\App\Http\Controllers\TaskCommentController::class, 'store']);
-Route::post('comment', [\App\Http\Controllers\CommentController::class, 'store']);
+Route::post('comments', [\App\Http\Controllers\CommentController::class, 'store']);
```

Comment:
Adding comment to a task is meant to be with respect to a specific task. I don't see how you are targeting the specific task that you want to add comments to.

Reply:
The post contains the user Id as well as the task id, the user clicks on the task he wants to comment on and in the URL you have taskid/comment this taskid is sent with the post data